reasoner/Reasoner.java

```diff
@@ -65,8 +65,7 @@ public Reasoner(TraversalEngine traversalEng, ConceptManager conceptMgr, LogicMa
 
     public ResourceIterator<ConceptMap> execute(Disjunction disjunction, boolean isParallel) {
         Set<Conjunction> conjunctions = disjunction.conjunctions().stream()
-                .map(conjunction -> logicMgr.typeResolver().resolveLabels(conjunction))
-                .map(conjunction -> logicMgr.typeResolver().resolveVariablesExhaustive(conjunction))
+                .map(conjunction -> logicMgr.typeResolver().resolveVariables(conjunction))
```

Comment:
`resolveVariables` now always calls `resolveLabels`. So there is no need to do both. This is because you always have to call `resolveLabels` for `resolveVariables` to run, so it shouldn't be the responsibility if the caller to ensure this.

Reply:
when then you should just rename the method `.resolveVariables()` to just `.resolve()`, no?